### PR TITLE
Add exercise: rotational-cipher

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

**Comment by SaschaMann** [comment]
Should be squashed on PR to remove the empty WIP commit

File: exercises/rotational-cipher/runtests.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+end
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# Additional exercises                                                #
+# Remove the comments for the optional bonus exercises from HINTS.md  #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Bonus A
+# @testset "string literal R13" begin
+#     @test R13"The quick brown fox jumps over the lazy dog." == "Gur dhvpx oebja sbk whzcf bire gur ynml qbt."
```
Comment (aawray) [inline exercises/rotational-cipher/runtests.jl]: You can use @test_skip as it is used in other language's test sets, just to conform common style (may it be?).

Comment (aawray) [inline exercises/rotational-cipher/runtests.jl]: Oh, just saw the issue regarding @test_skip and Travis.

Comment (SaschaMann) [inline exercises/rotational-cipher/runtests.jl]: I will try it but it threw an Error anyway on my end, so I thought Travis will fail the build.

Comment (SaschaMann) [inline exercises/rotational-cipher/runtests.jl]: Indeed, it throws a LoadError and fails the build.

Comment (andrej-makarov-skrt) [inline exercises/rotational-cipher/runtests.jl]: @test_skip behavior is a bit confusing, there must be discussion or we should make another macro for test skipping like @skip_test :)

Comment (SaschaMann) [inline exercises/rotational-cipher/runtests.jl]: Good idea, I'll leave this PR open until we have a solution to #45

**Comment by SaschaMann** [comment]
I will merge this now with the commented out tests and will revisit it in v0.6 once `@test_skip` behaves properly. Would be good to get some user feedback on how they'd like optional tests to be handled, too.

**Pull request merged**